wext/saddlepoint.py

```diff
@@ -5,16 +5,29 @@
 import itertools
 from constants import *
 
-def condition(state):
+def check_condition(state, condition):
 
     # Define a success condition for each state, e.g., mutual exclusivity.
 
-    if sum(state)==1:
-        return True
+    if condition=='exclusivity':
```

Comment:
Would prefer if these conditions were constants (e.g. `EXCLUSIVITY='exclusivity'` and `if condition == EXCLUSIVITY`).

Reply:
Done.  Note that constants add an additional step when users to define their own test statistics.